app/javascript/providers/UserProvider.js

```diff
@@ -13,11 +13,39 @@ export function getAccessToken() {
   return ('SCP' in window) ? window.SCP.userAccessToken : 'test'
 }
 
+/**
+ * Returns the current read-access token for the user
+ * This token is specifically for genome visualizations like IGV and Ideogram
+ * as they both require streaming a GCS object directly to the client
+ * This token needs the devstorage.readonly OAuth scope, which is returned
+ * from ApplicationHelper#get_read_access_token and stored in the window.SCP object
+ */
+export function getReadAccessToken() {
+  return ('SCP' in window) ? window.SCP.readOnlyToken : 'test'
+}
```

Comment:
Let's either call the function `getReadOnlyToken`, or call the thing it gets `readAccessToken`, for consistency.  The backend calls it "read access token" instead of "read only token", so I slightly prefer the former.

```suggestion
export function getReadAccessToken() {
  return ('SCP' in window) ? window.SCP.readAccessToken : 'test'
}
```

(This would be a global find-replace, of course.)